Dashboard/app/js/lib/views/reports/export-reports-views.js

```diff
@@ -187,17 +229,28 @@ FLOW.ExportReportsAppletView = FLOW.View.extend({
   }.property('FLOW.selectedControl.selectedQuestion'),
 
   showLastCollection: function () {
-    return FLOW.Env.showMonitoringFeature && FLOW.selectedControl.selectedSurveyGroup && FLOW.selectedControl.selectedSurveyGroup.get('monitoringGroup');
-  }.property('FLOW.selectedControl.selectedSurveyGroup'),
+    if (FLOW.selectedControl.selectedSurveyGroup && FLOW.selectedControl.selectedSurvey) {
+      //if not a monitoring form, export should be filtered by date
+      if (FLOW.selectedControl.selectedSurvey.get('keyId') == FLOW.selectedControl.selectedSurveyGroup.get('newLocaleSurveyId')) {
+        FLOW.ReportLoader.set('cleaningExportOption', "range");
+        $('input:radio[name=cleaning-export-option]').filter('[value=range]').prop('checked', true);
+        FLOW.ReportLoader.set('analysisExportOption', "range");
+        $('input:radio[name=analysis-export-option]').filter('[value=range]').prop('checked', true);
+      }
+    }
+    return FLOW.Env.showMonitoringFeature && FLOW.selectedControl.selectedSurveyGroup
```

Comment:
We have all instances running monitoring features now so we actually don't  need to check this anymore. `FLOW.ENV.showMonitoringFeature`